Chip 2 Final submission

gillrobyn opened the pull request:
I have removed the old files I had previously submitted, and have submitted 2 versions of Chip2 as well as an updated draft report. On my end it does not appear there are any outstanding verification issues.

gillrobyn committed:
Add files via upload

gillrobyn committed:
Merge branch 'SiEPIC:main' into main

gillrobyn committed:
Chip1DraftReport

gillrobyn committed:
Add files via upload

gillrobyn committed:
Delete submissions/ELEC413_robyng_project2_v01.gds

gillrobyn committed:
Add files via upload

gillrobyn committed:
Delete submissions/ELEC413_robyng_project2_v00.gds

gillrobyn committed:
Merge branch 'SiEPIC:main' into main

gillrobyn committed:
Merge branch 'SiEPIC:main' into main

gillrobyn committed:
Delete submissions/ELEC413_robyng_Project2_DraftReport_v02.docx

Delete report, will upload new version

gillrobyn committed:
Delete submissions/ELEC413_robyng_Project2_DraftReport.docx

Delete report, new version will be uploaded

gillrobyn committed:
Delete submissions/ELEC413_robyng_project2_v02.gds

Delete layout, new version will be uploaded

gillrobyn committed:
Chip 2 submission (2 chips + draft report)

Hang-Bobby-Zou commented:
Please fix the top cell name as well as opt_in label, the labels are the same for v03 and v04. Can I also ask why do you have identical designs? What is the purpose of it?

gillrobyn commented:
Can you clarify, what should the TOP cell name be? 

v03 has three identical MZIs to study the MFG tolerance across the chip
v04 has one target MZI plus 2 variants that have +/- 1% length difference in delta L

Hang-Bobby-Zou commented:
TOP cell should be named: ELEC413_<name> or something. 
v03: you are not going to get too useful of a data since they are very close to each other
v04: ok

gillrobyn committed:
Delete submissions/ELEC413_robyng_project2_v03.gds

gillrobyn committed:
Delete submissions/ELEC413_robyng_project2_v04.gds

gillrobyn committed:
Add files via upload

gillrobyn commented:
Thank you for all the feedback, I have updated both layout files and deleted the old ones.
I believe I have corrected the opt_in labels to all be unique across both files
I believe the TOP cell is now named correctly for each file
I have spaced out the MZIs in v03 to get a better idea of performance vs. placement on the chip.
Please let me know if you have additional feedback.

lukasc-ubc commented:
you have a v03 and v04 file.  Do you want to fabricate both?  Or just one.  
I'll merge it, but you can remove one in a new PR.

gillrobyn commented:
Yes! The intent is to fabricate both versions if that's okay, they are slightly different. Thanks.
